### PR TITLE
Add Gradle plugin integration tests

### PR DESCRIPTION
**Pull request by AlexeyTsvetkov**

### CODE REVIEW

File: gradle-plugins/compose/build.gradle.kts [hunk 1]
```diff
@@ -54,3 +56,25 @@ val jar = tasks.named<Jar>("jar") {
     from(zipTree(shadow.get().archiveFile))
     this.duplicatesStrategy = DuplicatesStrategy.INCLUDE
 }
+
+val gradleVersionForTests = "6.7"
+val java14Home: String? = when (JavaVersion.current()) {
+    JavaVersion.VERSION_14 -> System.getProperty("java.home")
```
Comment (olonho) [inline gradle-plugins/compose/build.gradle.kts:62]: As discussed, let's make it >=.

File: gradle-plugins/compose/src/test/test-projects/application/jvm/settings.gradle [hunk 2]
```diff
@@ -0,0 +1,11 @@
+pluginManagement {
+    plugins {
+        id 'org.jetbrains.kotlin.jvm' version '1.4.10'
```
Comment (olonho) [inline gradle-plugins/compose/src/test/test-projects/application/jvm/settings.gradle:3]: Interesting, why 1.4.10? Maybe 1.4.20 (latest) or 1.4.0 (used in Compose)?

**Review by olonho — COMMENTED** [review]
And as discussed, let's specify smallest supported Gradle version, and in test in more versions.

File: gradle-plugins/compose/src/test/test-projects/application/jvm/settings.gradle (hunk 2 above)
Comment (AlexeyTsvetkov) [reply gradle-plugins/compose/src/test/test-projects/application/jvm/settings.gradle]: Fixed

**Review by olonho — APPROVED** [review]

**Pull request merged**